mlir/tools/mlir-miopen-lib/CMakeLists.txt

```diff
@@ -97,4 +90,56 @@ if( BUILD_FAT_LIBMLIRMIOPEN )
   install(TARGETS MLIRMIOpen
     ARCHIVE DESTINATION lib)
 
+  # Install Miir.h to ${CMAKE_INSTALL_PREFIX}/include/<project-name>-<version>/
+  # as part of component libMLIRMIOpen
+  set(PACKAGE_DIR MLIRMIOpen)
+  install(FILES Miir.h
+    DESTINATION include/${PACKAGE_DIR}
+    COMPONENT libMLIRMIOpen EXCLUDE_FROM_ALL)
+
+  # Install lib${target_name} as part of ${component_name} and export it to be searchable by find_package()
+  function(export_target target_name component_name)
```

Comment:
Could you give a summary of what artifacts it install to file system? Give each entry a location and a brief description on why the file is needed and what it does?

Reply:
Actually for the review purpose. Would you mind to actually attach all of those files as a artifact to github? I want to take a look at all exported files.